### PR TITLE
💄 feat: minor fixes in design

### PR DESCRIPTION
**Pull request by pferdlexxie**

### CODE REVIEW

File: src/lib/components/Team.svelte [hunk 1]
```diff
@@ -36,7 +36,7 @@
             name: 'Ivan',
             surname: 'Golov',
             position: 'Figma master',
-            bio: 'I am a daisyUI master and Taiwind CSS lover',
+            bio: 'CSS master slave',
```
Comment (cuprum-acid) [inline src/lib/components/Team.svelte:39]: Bro is just some slave master ⛓️

File: src/lib/components/Team.svelte [hunk 2]
```diff
@@ -36,7 +36,7 @@
             name: 'Ivan',
             surname: 'Golov',
             position: 'Figma master',
-            bio: 'I am a daisyUI master and Taiwind CSS lover',
```
Comment (cuprum-acid) [inline src/lib/components/Team.svelte:39]: No more daisyUI and Tailwind CSS

File: src/lib/components/Transaction.svelte [hunk 3]
```diff
@@ -36,7 +36,7 @@
     <td>{stringDate}</td>
     <th>
         <a href={transactionPath}>
-            <button class="btn btn-ghost btn-xs">edit</button>
+            <button class="btn btn-ghost btn-s">edit</button>
```
Comment (cuprum-acid) [inline src/lib/components/Transaction.svelte:39]: Nice 😄

File: src/lib/components/TransactionView.svelte [hunk 4]
```diff
@@ -3,52 +3,57 @@
     export let date: Date;
     export let amount: number;
     export let category: string;
+    console.log(date);
+    console.log(category);
+
+    // data converter
+    let dateObject = new Date(date);
+    let year = dateObject.getFullYear();
+    let month = (dateObject.getMonth() + 1).toString().padStart(2, '0'); // Add 1 to month since it's zero-based
```
Comment (cuprum-acid) [inline src/lib/components/TransactionView.svelte:12]: Looks preety reliable 👀

File: src/lib/components/TransactionView.svelte [hunk 5]
```diff
@@ -57,24 +62,22 @@
...
-                </div>
+                <!-- <input class="join" /> -->
+                <select
+                    class="select input-bordered join-item mr-4 w-56 text-base font-medium"
+                    value={category}
+                >
+                    <option disabled selected>Category</option>
+                    <option>Food</option>
+                    <option>Daily</option>
+                    <option>Minibar</option>
```
Comment (cuprum-acid) [inline src/lib/components/TransactionView.svelte:73]: Good & reliable choise of drink! 🍺

**Review by cuprum-acid — APPROVED** [review]
Good & reliable minor fixes. I appreciate it.

**Review by axonstan — APPROVED** [review]

**Pull request merged**